packages/mongodb-chatbot-server/src/services/mongodbConversations.ts

```diff
@@ -126,6 +127,39 @@ export function makeMongoDbConversationsService(
       }
       return true;
     },
+
+    async commentMessage({
+      conversationId,
+      messageId,
+      comment,
+    }: CommentMessageParams) {
+      const updateResult = await conversationsCollection.updateOne(
+        {
+          _id: conversationId,
+          messages: {
+            $elemMatch: {
+              id: messageId,
+              role: "assistant",
+              rating: { $exists: true },
+              userComment: { $exists: false },
```

Comment:
think it could make sense to not have this as well so that a message can be updated even if it already exists